ItalianTranslation_PlanetCoaster.xml

```diff
@@ -4447,11 +4447,11 @@
 		<entry key="AdvancedTriggerEditor_DisplayTriggersLabel" translation="Inneschi Display"/>
 		<entry key="AdvancedTriggerEditor_DoneButton" translation="Fatto"/>
 		<entry key="AdvancedTriggerEditor_Duplicate" translation="Duplica"/>
-		<entry key="AdvancedTriggerEditor_Duration" translation="Duration"/>
+		<entry key="AdvancedTriggerEditor_Duration" translation="Durata"/>
 		<entry key="AdvancedTriggerEditor_EditTriggerSequenceButton" translation="Modifica Sequenza di Innesco"/>
-		<entry key="AdvancedTriggerEditor_FadeToColour" translation="Fade to Colour"/>
-		<entry key="AdvancedTriggerEditor_FadeToDuration" translation="Fade Time"/>
-		<entry key="AdvancedTriggerEditor_FadeToVolume" translation="Fade to Volume"/>
+		<entry key="AdvancedTriggerEditor_FadeToColour" translation="Dissolvenza per Colore"/>
+		<entry key="AdvancedTriggerEditor_FadeToDuration" translation="Dissolvenza Tempo"/>
```

Comment:
Credo che questa non vada bene, "Dissolvenza Tempo" non vuol dire molto, credo sia qualcosa più "Durata dissolvenza", hai cercato in-game dove appare?
In caso cerca e poi mi mandi il fix alla prossima pull

Reply:
Verifico